conf/settings.py

```diff
@@ -957,9 +899,4 @@
 GOOGLE_ANALYTICS_ID = None
 SENTRY_DSN = None
 SENTRY_ENVIRONMENT = SITE_DOMAIN
-AKISMET_API_KEY = None
-
-try:
-    from .local_settings import *
-except ImportError:
-    pass
```

Comment:
We should keep this line, it was a hack so users can define a new conf, and we can just replace the conf

Reply:
Oh okay thanks! I thought it was just removed by Weblate. I will readd it in packaging V2 then, and describe it was added by us for Yunohost.

Reply:
Good idea!
That's the main point of my review

Reply:
Thank you, I appreciate!
FYI, there is a little bug in master because I forgot to add the /cache/ folder (before static) which is solved in V2